apps/demo/src/app/cart/components/cart-wrapper/cart-wrapper.component.spec.ts

```diff
@@ -139,8 +133,8 @@ describe('CartWrapper', () => {
       cartComponent = fixture.debugElement.query(By.css('demo-cart'));
       cartTotalsComponent = fixture.debugElement.query(By.css('demo-cart-totals'));
       helpBoxComponent = fixture.debugElement.query(By.css('demo-help-box'));
-      proceedToCheckoutComponent = fixture.debugElement.query(By.css('[demo-proceed-to-checkout]'));
-      continueShoppingComponent = fixture.debugElement.query(By.css('[demo-continue-shopping]'));
+      proceedToCheckoutComponent = fixture.debugElement.query(By.css('[demoProceedToCheckout]'));
+      continueShoppingComponent = fixture.debugElement.query(By.css('[demoContinueShopping]'));
```

Comment:
should be continueShoppingDirective and proceedToCheckoutDirective

Reply:
resolved in 0125e29